docs/source/dependencies/seqrepo.rst

```diff
@@ -0,0 +1,190 @@
+    tar -xzvf seqrepo_2024-12-20.tar.gz -C seqrepo
+
+3. Set the Environment Variable
+-------------------------------
+
+Configure your environment to point to this SeqRepo location by setting the following variable in your `.env` file: ::
+
+    SEQREPO_DATAPROXY_URI="seqrepo+file:///full/path/to/seqrepo/2024-12-20"
+
+Replace ``/full/path/to/seqrepo`` with your absolute path.
+
+Docker-based Installation
+=========================
+
+Docker is suitable for containerized environments or users preferring isolation.
+
+Prerequisites
+-------------
+
+* Ensure Docker and Docker Compose are installed.
+
+  * `Docker Installation <https://docs.docker.com/get-docker/>`_
+  * `Docker Compose Installation <https://docs.docker.com/compose/install/>`_
+
+1. Create a Docker Volume
+-------------------------
+
+::
+
+    docker volume create seqrepo-vol
+
+2. Populate the Volume With a SeqRepo Snapshot
+----------------------------------------------
+
+::
+
+	docker run -it --rm -v seqrepo-vol:/usr/local/share/seqrepo docker.io/biocommons/seqrepo:2024-12-20
+
+.. NOTE::
+
+    This downloads and syncs the SeqRepo database into the volume. It may take a while.
+
+3. Pull the SeqRepo REST Service Image
+--------------------------------------
+
+::
+
+    docker pull biocommons/seqrepo-rest-service
+
+4. Start SeqRepo REST Service
+-----------------------------
+
+Run the REST service to provide access through HTTP. Choose an appropriate local port - the example below uses ``5001``, but you can pick anything you'd like. ::
+
+    docker run -d --name seqrepo-rest \
+    -v seqrepo-vol:/usr/local/share/seqrepo \
+    -p 5001:5000 biocommons/seqrepo-rest-service \
+    seqrepo-rest-service /usr/local/share/seqrepo/2024-12-20
+
+5. Set your environment variable to the REST API
+------------------------------------------------
+
+Set the following environment variables in your ``.env`` file: ::
+
+    SEQREPO_DATAPROXY_URI="seqrepo+http://localhost:5001/seqrepo"
+    SEQREPO_INSTANCE_DIR="/usr/local/share/seqrepo/2024-12-20"
+
+Native Installation (rsync-based)
+==================================
+
+Useful when direct file or Docker setups are not feasible.
+
+1. Install SeqRepo
+------------------
+
+::
+
+    pip install seqrepo
+
+2. Prepare directories
+----------------------
+
+::
+
+    export SEQREPO_VERSION=2024-12-20
+    sudo mkdir -p /usr/local/share/seqrepo
+    sudo chown $USER /usr/local/share/seqrepo
+
+3. Fetch data with ``rsync``
+----------------------------
+
+::
+
+    seqrepo pull -i $SEQREPO_VERSION
+    seqrepo update-latest
+
+4. Configure environment
+------------------------
+
+Set the environment variable in your ``.env`` file accordingly:
+
+::
+
+    SEQREPO_DATAPROXY_URI=seqrepo+file:///usr/local/share/seqrepo/$SEQREPO_VERSION
+
+Troubleshooting
+---------------
+
+* If your institution blocks rsync, consider using the direct download method or Docker.
+* Ensure you have the correct `rsync` executable: GNU rsync, NOT openrsync.
```

Comment:
I know there is a warning message in seqrepo to use --rsync-exe, but non-technical users may not know how to find the location. Might be good to include an example `seqrepo --rsync-exe $(which rsync)`

Reply:
^assuming the standard rsync was installed with `brew`